src/Samples/Masa.Maui/Data/App/ECommerce/Dto/BasketItemDto.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Masa.Maui.Data.App.ECommerce.Dto;
+
+public class BasketItemDto
+{
+    public BasketItemDto(int id, string name, string company, float score, int qty, string delivery,
+        string offers, decimal price, string pictureFileName, bool freeShipping)
+    {
```

Comment:
Here is format issues need be fixed ☕ 
```suggestion
    string offers, decimal price, string pictureFileName, bool freeShipping)
    {
```